src/mpyl/reporting/targets/jira.py

```diff
@@ -58,7 +58,7 @@ def from_issue_response(response):
         ticket_id = response['key']
         assignee = fields.get('assignee')
         user = assignee or fields.get('reporter') or fields.get('creator')
-        avatar = user['avatarUrls']['24x24'] if user else ''
+        avatar = user['avatarUrls']['24x24'] if user and user['avatarUrls']['24x24'].endswith('24') else ''
```

Comment:
We can also try to set the avatar in `to_markdown_summary` with an HTML tag 
`f'<img src="{avatar}" width="24" height="24">'`